Makefile

```diff
@@ -179,6 +179,11 @@ dependencies-debian:  ## install dependencies for linux
 dependencies-fedora:  ## install dependencies for linux
 	yum install -y automake bison ccache cmake curl flex perl-IPC-Cmd tar unzip zip
 
+dependencies-alma:  ## install dependencies for linux
```

Comment:
alma is redhat/fedora derived, it should not have a separate target

Reply:
I added separate target to install epel and ensure ccache gets installed.

How would you suggest I do that without adding a new makefile target? Note that the existing `dependencies-fedora` target will fail on the manylinux_2_28 containers because epel isn't configured.

Reply:
that should work fine for other redhat/fedora flavors. Certainly better than adding `4000` makefile entries for every distro.